WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java

```diff
@@ -392,6 +393,20 @@ private void startWebActivity(String themeId, ThemeWebActivity.ThemeWebActivityT
         if (NetworkUtils.isNetworkAvailable(this)) {
             if (mCurrentTheme != null) {
                 boolean isCurrentTheme = mCurrentTheme.getId().equals(themeId);
+                switch (type) {
+                    case PREVIEW:
+                        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.THEMES_PREVIEWED_SITE);
+                        break;
+                    case DEMO:
+                        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.THEMES_DEMO_ACCESSED);
+                        break;
+                    case DETAILS:
+                        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.THEMES_DETAILS_ACCESSED);
+                        break;
+                    case SUPPORT:
+                        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.THEMES_SUPPORT_ACCESSED);
+                        break;
+                }
```

Comment:
As above, for all of these calls can we please add the `blog_id` and the `theme_id` to properties?

See https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java for details. 

note: Make sure that the "remote blog id" is correctly used it. `String blogID = StatsUtils.getBlogId(blog);` See the class above for details ;)